cs/v1/CS_v1.0.0.yaml

```diff
@@ -770,7 +803,7 @@ components:
           example: CY
         cutOffTimes:
           type: array
-          description: A list of cut-offs times provided by the carrier when available. A cut-off time indicates the latest deadline within which a task must be completed.
+          description: A list of cut-offs times provided by the carrier when available. A cut-off time indicates the latest date and time by which a task must be completed. For example, the latest test date and time by which a container must be delivered to a terminal to be loaded on a vessel, or where certain documentation must be provided by the Shipper.
```

Comment:
Wondering about the term "...For example, the latest **test** date and...". What does `test` date mean in this context?

Reply:
It was a typo